spec/presenters/edition_presenter_spec.rb

```diff
@@ -36,6 +36,10 @@
     it "mixes in the specified update_type to the presentation" do
       expect(subject[:update_type]).to eq update_type
     end
+
+    it "adds the supertypes" do
+      expect(subject["user_journey_document_supertype"]).to eq "thing"
```

Comment:
@kevindew I understood it as coming from https://github.com/alphagov/govuk_document_types/blob/577152696c4554e7096f595016cea400b2c3397f/data/supertypes.yml#L34